recipe/meta.yaml

```
@@ -38,27 +36,28 @@ outputs:
         - python
         - pip
         - cython >=0.29,<3.1
-        - numpy >=1.21  # [not ppc64le]
-        - numpy >=1.19  # [ppc64le]
+        - numpy 1.21  # [not ppc64le or py>=310]
```

This also won't work with 3.11

---

sure, we need to synchronize numpy to get rid of such pinnings

---

I would set this instead:
```suggestion
        - numpy 1.21  # [not ppc64le or py==310]
        - numpy       # [not ppc64le or py>=311]
```